### PR TITLE
Homescreen Contact Diary card (EXPOSUREAPP-4153)

### PR DESCRIPTION
**Pull request by SamuraiKek**
This PR adds the new Contact Diary card to the homescreen.

### CODE REVIEW

File: Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/main/MainActivity.kt [hunk 1]
```diff
@@ -197,6 +198,11 @@ class MainActivity : AppCompatActivity(), HasAndroidInjector {
         onBackPressed()
     }
 
+    fun navigateToContactDiaryActivity() {
+        ContactDiaryActivity.start(this)
+        finish()
```
Comment (d4rken) [inline Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/main/MainActivity.kt:203]: Shouldn't the user get back the to main CWA activity after exiting the diary?

**Review by janetback — APPROVED** [review]
UA approved

File: Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/main/MainActivity.kt (hunk 1 above)
Comment (SamuraiKek) [reply Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/main/MainActivity.kt:203]: Ah yes you're right :) I forgot to remove this before pushing.

File: Corona-Warn-App/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -1489,4 +1489,11 @@
...
     <string name="submission_test_result_positive_no_consent_button_abort">"Cancel"</string>
 
+    <!-- XHED: Title for the contact diary card displayed in the homescreen -->
+    <string name="contact_diary_homescreen_card_header" />
+    <!-- XTXT: Body for the contact diary card displayed in the homescreen  -->
+    <string name="contact_diary_homescreen_card_body" />
+    <!-- XBUT: Button text for the contact diary card button in the homescreen   -->
+    <string name="contact_diary_homescreen_card_button_text" />
+
 </resources>
```
Comment (kolyaopahle) [inline Corona-Warn-App/src/main/res/values/strings.xml:1499]: \n

File: Corona-Warn-App/src/main/res/layout/contact_diary_homescreen_card_include.xml [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+            app:srcCompat="@drawable/ic_diary_book" />
+
+        <androidx.constraintlayout.widget.Barrier
+            android:id="@+id/submission_status_card_unregistered_barrier"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            app:barrierDirection="bottom"
+            app:constraint_referenced_ids="contact_diary_card_homescreen_icon,contact_diary_card_homescreen_body" />
+    </androidx.constraintlayout.widget.ConstraintLayout>
+</layout>
```
Comment (kolyaopahle) [inline Corona-Warn-App/src/main/res/layout/contact_diary_homescreen_card_include.xml:74]: \n

File: Corona-Warn-App/src/main/res/drawable/ic_diary_book.xml [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+            android:pathData="M64.4999 108.467H12.5099C7.56303 108.467 3.56543 104.432 3.56543 99.4619V98.4212C3.56543 93.4406 7.57358 89.4159 12.5099 89.4159H64.4893C69.4362 89.4159 73.4338 93.4513 73.4338 98.4212V99.4619C73.4444 104.432 69.4362 108.467 64.4999 108.467Z"
+            android:fillColor="#FFFFFF" />
+        <path
+            android:pathData="M85.817 108.467H13.0269V89.4055H85.817C85.0892 90.6374 83.2644 94.0675 83.2644 98.9312C83.2644 100.418 83.4332 104.443 85.817 108.467Z"
+            android:fillColor="#FFFFFF" />
+        <path
+            android:pathData="M13.9231 99.345H24.4287C24.7451 99.345 24.9983 99.5999 24.9983 99.9184V119.426C24.9983 120 24.2599 120.223 23.9435 119.745L19.6505 113.256C19.4185 112.917 18.9227 112.917 18.7012 113.256L14.4083 119.745C14.0919 120.223 13.3535 120 13.3535 119.426V99.9184C13.3535 99.5999 13.6067 99.345 13.9231 99.345Z"
+            android:fillColor="#657887" />
+    </group>
+</vector>
```
Comment (kolyaopahle) [inline Corona-Warn-App/src/main/res/drawable/ic_diary_book.xml:22]: \n

File: Corona-Warn-App/src/main/res/values/strings.xml (hunk 2 above)
Comment (SamuraiKek) [reply Corona-Warn-App/src/main/res/values/strings.xml:1499]: I thought we could just add the string names in here but empty. Haven't had complaints so far :D

File: Corona-Warn-App/src/main/res/drawable/ic_diary_book.xml (hunk 4 above)
Comment (SamuraiKek) [reply Corona-Warn-App/src/main/res/drawable/ic_diary_book.xml]: Added new line.

File: Corona-Warn-App/src/main/res/layout/contact_diary_homescreen_card_include.xml (hunk 3 above)
Comment (SamuraiKek) [reply Corona-Warn-App/src/main/res/layout/contact_diary_homescreen_card_include.xml]: Added new line.

**Pull request merged**